vite.config.ts

```diff
@@ -10,6 +10,7 @@ import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfil
 
 export default defineConfig({
   plugins: [preact(), tsconfigPaths()],
+  resolve: { alias: { assert: 'assert-browserify' } },
```

Comment:
We should check if removing `assert` from `inject` below breaks proof generation or minting